src/OpenLoco/Input/ShortcutManager.cpp

```diff
@@ -89,6 +99,15 @@ namespace OpenLoco::Input::ShortcutManager
         { makeScreenshot,              StringIds::shortcut_screenshot,                     "makeScreenshot",              "Left Ctrl+S" },
         { toggleLastAnnouncement,      StringIds::shortcut_toggle_last_announcement,       "toggleLastAnnouncement",      "Space" },
         { sendMessage,                 StringIds::shortcut_send_message,                   "sendMessage",                 "F1" },
+        { constructionPreviousTab,        StringIds::shortcut_construction_previous_tab,          "constructionPreviousTab",        "" },
```

Comment:
The table needs to be realigned, and perhaps reordered a little. (The new serialization feature introduced in #1141 allows for the order to change.)